docs/build_options.md

```diff
@@ -10,6 +10,7 @@ You should set those while invoking CMake to build the project. See the [example
 | `USE_QT_VERSION`                           | (unset)       | `5` / `6`           | Use a specific version of Qt to build the app               |
 | `GIT_REVISION`                             | `OFF`         | `ON` / `OFF`        | Append the current git revision to the app's version string |
 | `UPDATE_CHECKER`                           | `ON`          | `ON` / `OFF`        | Enable or disable both the update checker and auto-updater  |
+| `PRO_VERSION`                              | `OFF`         | `ON` / `OFF`        | Enable or disable Notes Pro features                        |
```

Comment:
This is now `ON` by default.

Reply:
Yes, as you suggested, right?

Reply:
Yes, but the default value needs to be updated in this documentation as well (it's still `OFF`). 😄

Reply:
Ohh right hah